Lib/test/test_email/test_message.py

```diff
@@ -1004,6 +1004,34 @@ def test_folding_with_long_nospace_http_policy_1(self):
         parsed_msg = message_from_bytes(m.as_bytes(), policy=policy.default)
         self.assertEqual(parsed_msg['Message-ID'], m['Message-ID'])
 
+    def test_no_wrapping_with_zero_max_line_length(self):
```

Comment:
AFAIK, those two tests are identical. If so, create a function taking the maxlinelen, say `do_test_no_wrapping_max_line_length(n)` and call it for 0 and None.